static/app/components/events/contexts/index.tsx

```diff
@@ -16,6 +15,38 @@ type Props = {
   group?: Group;
 };
 
+type ContextValueType = any;
+
+export function getOrderedContextItems(event): [string, ContextValueType][] {
```

Comment:
nit: not that big of a deal but returning an array of objects here would be a little more descriptive since you can name the return values.